sensor/common/deduper/key_test.go

```diff
@@ -0,0 +1,119 @@
+package deduper
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/fixtures/fixtureconsts"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	randomID   = "2188759360372708523"
+	randomHash = 2188759360372708523
```

Comment:
These don't seem random :-) 

How about `StubID` and `StubHash`?

Reply:
Done